src/main/java/core/basesyntax/FileWork.java

```diff
@@ -17,7 +17,7 @@
  */
 public class FileWork {
 
-    static String FORMATER = "w";
+    public final String formater = "w";
```

Comment:
не забывай о том, как нужно оформлять константы
они должны быть `static final`, название информативное и в верхнем регистре
```suggestion
    public static final String FIRST_CHARACTER = "w";
```

Reply:
З першого разу не зрозумів ))) Пофіксив.